File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    
    OX_lst = []
    for q in quiz:
        one_quiz = q.split()
        
        if one_quiz[1] == '+':
            if int(one_quiz[0]) + int(one_quiz[2]) == int(one_quiz[4]):
                OX_lst.append("O")
            else:
                OX_lst.append("X")
                
        elif one_quiz[1] == '-':
            if int(one_quiz[0]) - int(one_quiz[2]) == int(one_quiz[4]):
                OX_lst.append("O")
            else:
                OX_lst.append("X")
    
    return OX_lst

print(solution(["3 - 4 = -3", "5 + 6 = 11"]))

File: 프로그래머스/unrated/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    let result = []
    
    for (let i = 0; i < quiz.length; i++){
        quiz[i] = quiz[i].split(" ")
        let X = Number(quiz[i][0]), Y = Number(quiz[i][2]), Z = Number(quiz[i][4])
        
        if (quiz[i][1] === "-"){
            if (X - Y === Z) { result.push("O")}
            else if (X - Y !== Z) { result.push("X")}
        }
        else if (quiz[i][1] === "+"){
            if (X + Y === Z) { result.push("O")}
            else if (X + Y !== Z) { result.push("X")}
        }
    }
    return result;
}